Toolchain.mk

```
@@ -53,8 +53,13 @@ $(CRDOC): | $(BIN)
 	$(info $(M) installing crdoc)
 	@{ \
 	set -e ;\
-	GOBIN=$(BIN) go install fybrik.io/crdoc@$(CRDOC_VERSION) ;\
-	mv $(BIN)/crdoc $(CRDOC) ;\
+	OSTYPE=$(shell uname | awk '{print tolower($$0)}') && ARCH=$(shell go env GOARCH) && \
+	if [ "`go env GOARCH`" = "amd64" ]; then ARCH="x86_64"; fi && \
+	curl -sSLfo $(CRDOC).tar.gz $(CURL_GH_AUTH) https://github.com/fybrik/crdoc/releases/download/$(CRDOC_VERSION)/crdoc_$${OSTYPE}_$${ARCH}.tar.gz && \
```

Even though it works, strictly speaking, this URL is not accurate, because crdoc uses title case for operating system names, GitHub just automatically redirects to the correct asset. I would still suggest to correct the code here.
```suggestion
	OSTYPE=$(shell uname) && ARCH=$(shell go env GOARCH) && \
	if [ "`go env GOARCH`" = "amd64" ]; then ARCH="x86_64"; fi && \
	curl -sSLfo $(CRDOC).tar.gz $(CURL_GH_AUTH) https://github.com/fybrik/crdoc/releases/download/$(CRDOC_VERSION)/crdoc_$${OSTYPE}_$${ARCH}.tar.gz && \
```